data.js

```diff
@@ -208,6 +208,37 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- literature wrote \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of [2018](https://www.wri.org/annualreport/2018-19#page=4) and their are (more)[https://www.wri.org/our-work/outcomes].",
```

Comment:
The language seems a bit off (`There are some of 2018 and their are more`). But more general I think the `actions` field should contain a description of their actions, not just links to reports or their website.

Reply:
put some real actions in there